opensir/models/model.py

```diff
@@ -39,12 +39,14 @@ def __init__(self):
         self.p = None
         self.pop = None
         self.w0 = None
-        # Attributes setted after fitting
-        self.pcov = None
         self.fit_input = None
         self.fit_index = None  # Store fitting info for post-regression analysis
-        self.t_obs = None
-        self.n_obs = None
+        # Attributes setted after fitting
+        self.postreg = {
```

Comment:
See https://github.com/open-sir/open-sir/pull/77/commits/20387251101e11f3ab8ec55436bc7b90bad701fa#r418971623.
IMO the only one that makes sense to keep here is "pcov", which could be a simple member

Reply:
ok, I see the spirit behind these (they are used by `ci_xxx` functions).

However, in this case I think I would drop the dict and leave it as it was before (simple members). IMO this adds unnecessary maintaining overhead

Reply:
pylint complains for too much attributs, that was my motivation of creating this new structure

Reply:
Adding a dict removes the pylint warning but the spirit behind pylint's complain is the same :) We are adding 3 new variables anyway.

I will remove `fit_input` and `fit_index` in a follow up (WIP), so pylint shouldn't complain here.

Reply:
OK, so should I wait your follow up?

Where are you going to store fit_input and fit_index?